### ISSUE TITLE
Problem with file size 

### ISSUE THREAD

**Issue by po1yarniy** [DominicTobias/react-image-crop#536]
Hello everyone
Perhaps someone has encountered a similar problem.

How to solve the problem, so that the format of the image after cropping does not change, and the file size does not increase?
input: jpeg file size 1920x1280 / 239kb
output: png file size 1917x1078 / 2.36mb

**Comment by IDavidCP** [comment]
I'm having the same issue. Changing to JPEG with `canvas.toBlob(callback, 'image/jpeg', 0.8)` does help, the file size won't go too high but the resolution is still increasing. For me setting 80% quality is the right balance. Thanks @DominicTobias for the solution 👍

**Comment by DominicTobias** [comment]
@IDavidCP is the image coming out larger than expected? Are you on a retina device like a macbook? If you put `window.devicePixelRatio` in your console what does it say? If the value is larger than `1` then the image will come out bigger than expected. I should put an example up of resizing the image down in an offscreen canvas before saving but I haven't written the code yet and been lazy to do it... will do at some point

**Comment by po1yarniy** [comment]
> Привет, PNG намного больше, чем формат JPG. Если вы посмотрите на [синтаксис toBlob](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob#syntax) (или toDataURL, хотя это не рекомендуется), он может принимать 3 аргумента, `toBlob(callback, type, quality)`поэтому вы можете `toBlob(callback, 'image/jpeg', 1)`также снизить качество jpeg, например, `toBlob(callback, 'image/jpeg', 0.8)`до 80%.

Thank you, it works!

**Comment by IDavidCP** [comment]
@DominicTobias Yes, the image is coming out larger than expected. I'm not using retina device and `window.devicePixelRatio` is `1`.  This is function I'm using to create the blob:
```typescript
function toBlob(canvas: HTMLCanvasElement): Promise<Blob | null> {
  return new Promise((resolve) => {
    canvas.toBlob(resolve, 'image/jpeg', 0.8)
  })
}
```
Original image:  .jpg 900x900 100kb
New image: .jpeg 1349x1349 141kb
